sympy/series/limits.py

```diff
@@ -162,8 +162,12 @@ def doit(self, **hints):
                                         for m in Mul.make_args(a))
                                     for a in Add.make_args(w)))
                 if all(ok(w) for w in e.as_numer_denom()):
-                    u = Dummy(positive=(z0 is S.Infinity))
-                    inve = e.subs(z, 1/u)
+                    if z0 is S.NegativeInfinity:
+                        u = Dummy(positive=True)
+                        inve = e.subs(z, -1/u)
+                    else:
+                        u = Dummy(positive=(z0 is S.Infinity))
```

Comment:
I am not sure. If the limit is in the real field, then it is not necessary.
If the limit is in the complex field, probably it is better to have such condition.